webauthn-server-core/src/test/scala/com/yubico/webauthn/RelyingPartyStartOperationSpec.scala

```diff
@@ -981,6 +982,39 @@ class RelyingPartyStartOperationSpec
         }
       }
 
+      it("allows setting the hints to a value not in the spec.") {
+        val pkcco = relyingParty(userId = userId).startRegistration(
+          StartRegistrationOptions
+            .builder()
+            .user(userId)
+            .hints("hej")
+            .build()
+        )
+        pkcco.getHints.asScala should equal(List("hej"))
+      }
+
+      it("allows setting the hints to a value in the spec.") {
+        val pkcco = relyingParty(userId = userId).startRegistration(
+          StartRegistrationOptions
+            .builder()
+            .user(userId)
+            .hints(PublicKeyCredentialHint.SECURITY_KEY)
+            .build()
+        )
+        pkcco.getHints.asScala should equal(List("security-key"))
+      }
+
+      it("allows setting the hints to empty") {
+        val pkcco = relyingParty(userId = userId).startRegistration(
+          StartRegistrationOptions
+            .builder()
+            .user(userId)
+            .hints("")
+            .build()
+        )
+        pkcco.getHints.asScala should equal(List(""))
+      }
+
```

Comment:
```suggestion

      it("omitted hints should result in empty list") {
        val pkcco = relyingParty(userId = userId).startRegistration(
          StartRegistrationOptions
            .builder()
            .user(userId)
            .build()
        )
        pkcco.getHints.asScala should equal(List())
      }
      
      it("allow setting multiple hints") {
        val pkcco = relyingParty(userId = userId).startRegistration(
          StartRegistrationOptions
            .builder()
            .user(userId)
            .hints(PublicKeyCredentialHint.SECURITY_KEY, PublicKeyCredentialHint.CLIENT_DEVICE)
            .build()
        )
        pkcco.getHints.asScala should equal(List("security-key", "client-device"))
      }
```

Just adding two more tests here that I have in my local / unfinished branch

Reply:
Thanks, but I had already added these but not pushed them yet.